src/EmbeddingSpMDMNBit.cc

```diff
@@ -737,34 +713,27 @@ typename ReturnFunctionSignature<indxType, offsetType, ROWWISE_SPARSE>::
                  ++i) {
               vec_reg_t out_vreg = vec_reg_t(v + i);
               if (i == 0) {
-                a->vpmovsxbd(temp_vreg, x86::Xmm(src_vreg.id()));
+                a->vpmovsxbd(temp_vreg, src_vreg.xmm());
               } else {
                 if (instSet == inst_set_t::avx512) {
                   // We could've used avx512_ymm for clock frequency advantage,
                   // if there's an instruction to extract a 64-bit portion from
                   // a YMM as an XMM register.
-                  a->vextracti32x4(
-                      x86::Xmm(temp_vreg.id()), src_vreg, asmjit::Imm(i));
+                  a->vextracti32x4(temp_vreg.xmm(), src_vreg, asmjit::Imm(i));
                 } else {
                   if (i == 1) {
-                    a->pextrq(
-                        scratchReg3_, x86::Xmm(src_vreg.id()), asmjit::Imm(1));
-                    a->movq(x86::Xmm(temp_vreg.id()), scratchReg3_);
+                    a->pextrq(scratchReg3_, src_vreg.xmm(), asmjit::Imm(1));
```

Comment:
I think there's bug in asmjit. The changes in these lines and lines 731-732 are causing invalid instruction errors. CC @kobalicek

Reply:
The bug been fixed, thanks!

Reply:
@jspark1105 We have asmjit fixed to a particular hash in OSS. We will have to update that. For internal, we will need to update asmjit in third-party.

Reply:
@dskhudia yeah let's defer landing this diff until we update asmjit next time. This diff can definitely wait.